src/extraction/wikidata.py

```diff
@@ -229,20 +235,33 @@ def _write_to_db(
     db_conn: sqlite3.Connection,
     title_to_id: Dict[str, str],
     id_to_attrs: Dict[str, Dict[str, Any]],
+    lang: str,
 ) -> None:
     """Persists qid information to database.
     db_conn (Connection): Database connection.
     title_to_id (Dict[str, str]): Titles to QIDs.
     id_to_attrs (Dict[str, Dict[str, Any]]): For QID a dictionary with property name to property value(s).
+    lang (str): Language.
     """
 
-    entities: List[Tuple[Optional[str], ...]] = []
+    entities: List[Tuple[str, int]] = []
     entities_texts: List[Tuple[Optional[str], ...]] = []
     aliases_for_entities: List[Tuple[str, str, int]] = []
+    # Text snippets indicating entity is a meta entity, i.e. a category or disambiguation page.
+    with open(
+        Path(__file__).parent.parent.parent / "configs" / "meta_terms.yaml", "r"
+    ) as stream:
+        meta_indicators = set(yaml.safe_load(stream)["entities"][lang])
 
     for title, qid in title_to_id.items():
         label = id_to_attrs[qid].get("labels", {}).get("value", None)
-        entities.append((qid,))
+        # Don't save meta entities.
+        if any(
+            [mi in id_to_attrs[qid].get("description", "") for mi in meta_indicators]
+        ):
+            continue
+
+        entities.append((qid, 0))
```

Comment:
I might not be interpreting this PR correctly, but I'm confused when `is_meta` would actually be `1` in the DB when we don't store them?

Reply:
The decision to not store meta entries was mine and never discussed. We could just as well decide to store meta entries, and ignore them during retrieval. Do you think it's fine if we decide to not store meta entries at all? Then we can drop `is_meta` completely.

Reply:
I was just confused because you had the whole framework to store `is_meta` but then not actually using it 😁 

Hmmm. It seems silly now to remove the functionality entirely and just never store them. Feature flag? 🙈

Reply:
Understandable :smile: I intended to store everything in the beginning, but then decided that there is no point in it.
Feature flag it is.

Reply:
Added in https://github.com/explosion/wikid/pull/16/commits/6866fc14c3e406de7f19db308f4a6b7cc1655e87.